pkg/startup/startup.go

```diff
@@ -687,11 +715,35 @@ func evaluateENVBool(envVar string, defaultValue bool) bool {
 
 // autoDetectCIDR auto-detects the IP and Network using the requested
 // detection method.
-func autoDetectCIDR(method string, version int) *cnet.IPNet {
+func autoDetectCIDR(method string, version int,
+	k8sClient kubernetes.Interface, nodeName string) *cnet.IPNet {
```

Comment:
IMO this can be a single line. We don't have line limitations in Calico and its easier to read on one line than on two.